decidim-admin/app/views/decidim/admin/organization/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :title do %>
+  <h2><%= t ".title" %></h2>
+<% end %>
+
+<%= form_for(@form, url: organization_path, format: :put) do |f| %>
```

Comment:
The `format` part is not needed, but since the route is a `resource` I need to specify the correct URL. Otherwise, if there's an error in the form, I get redirected to `admin/organization.1` and this sucks.